service/collector/chart_config_resource.go

```diff
@@ -37,36 +37,36 @@ var (
 	)
 )
 
-// ChartResourceConfig is this collector's configuration struct.
-type ChartResourceConfig struct {
+// ChartConfigResourceConfig is this collector's configuration struct.
+type ChartConfigResourceConfig struct {
```

Comment:
`ChartConfigResourceConfig` too redundant?

Reply:
When you want to make it consistent with the metrics, it would be `ChartConfigCordonExpireTime`.

Reply:
We have another metrics `chartconfig_status` so I think to change the name of `Config` struct as similar as just one metric doesn't quite look so good.